apps/src/applab/Visualization.jsx

```diff
@@ -0,0 +1,32 @@
+var applabConstants = require('./constants');
+
+var styles = {
+  hidden: {
+    display: 'none'
+  }
+};
```

Comment:
This is the third time this style is repeated in this PR.  It might be time for a small styles.js file.